pkg/nar/header.go

```diff
@@ -8,11 +8,11 @@ import (
 // Header represents a single header in a NAR archive. Some fields may not
 // be populated depending on the Type.
 type Header struct {
-	Type       EntryType // Typeflag is the type of header entry.
-	Name       string    // Name of the file entry
-	LinkTarget string    // Target of symlink (valid for TypeSymlink)
-	Size       int64     // Logical file size in bytes
-	Executable bool      // Set to true for files that are executable
+	Path       string   // Path of the file entry, relative inside the NAR
```

Comment:
If I remember correctly, this is the name of the entry. For example if you have a file under `bin/myexe`, you would have directory header with name "bin", and then a file with name "myexe". Path makes me think that the latter entry would contain `bin/myexe`.